src/settings.rs

```diff
@@ -57,6 +57,18 @@ Settings which only apply to certain [`Planner`](crate::planner::Planner)s shoul
 #[derive(Debug, serde::Deserialize, serde::Serialize, Clone)]
 #[cfg_attr(feature = "cli", derive(clap::Parser))]
 pub struct CommonSettings {
+    /// Attribute the diagnostics of this run to a specific usage category
+    #[cfg_attr(
+        feature = "cli",
+        clap(
+            long,
+            default_value = None,
+            env = "NIX_INSTALLER_ATTRIBUTION",
+            global = true
+        )
+    )]
+    pub attribution: Option<String>,
```

Comment:
Our existing diagnostic related argument has a `diagnostic-` prefix:

https://github.com/DeterminateSystems/nix-installer/blob/1c103edb9033eb4f954e14e9de94535b3f06c462/src/settings.rs#L208-L234

I think this one should too (`--diagnostic-attribution`) since it is only used in diagnostics.

Reply:
Done!